about-page.html

```diff
@@ -0,0 +1,112 @@
+            <h2 class="timing">We Try To Realize And Fulfil All Your Requirements</h2>
+            <p class="text-white company-name">Infinity Company</p>
+        </div>
+    </section>
+    <main class="main-about flex p-4 gap-2">
+        <h1 class="heading text-black mt-5">Few words about us</h1>
+        <hr class="hr-line">
+        <p class="about-p">Infinity is a privately owned website development and IT solutions company registered with AISA offering
+            various services to clients all over Afghanistan. We began our successful foray into the IT world in 2011
+            and are running parallel with the rapidly accelerating stream of technology ensuring that we stay at par
+            with the global changes that affect the business world.
+            Today we have achieved various laurels in providing services and became preferred vendor to many small to
+            large businesses all across the country. With our unique consulting approach, specialized developers team,
+            quick response to queries, affordable development costs and professional designs we have been successful in
+            creating various differentiators in this emerging market.</p>
+        <img src="./assets/images/slider-01.jpg" alt="">
+    </main>
+    <hr>
+    <section class="past flex p-4 gap-4 my-5">
+        <h1 class="heading text-black">See the Projects
+            <hr class="hr-line mt-2">
+        </h1>
+        <p class="about-p">We have done 500+ websites, almost every industry around the world. All of our Designers and Developers are
+            young and have enthusiasm oozing from their eyes. here are few recent works</p>
+        <div class="main-cards flex gap-5 flex-md-row">
+            <div class="first flex">
+                <p class="h1">2020</p>
+                <p>Thebookbazaar.af</p>
+            </div>
+            <div class="second flex">
+                <p class="h1">2021</p>
+                <p>Product Landing</p>
+            </div>
+        </div>
+    </section>
+
+    <footer>
+        <div class="partners">
+            <h1 class="heading">Partner</h1>
+            <hr class="hr-line">
+            <ul class="d-flex justify-content-center align-items-center flex-wrap gap-3">
+                <li><img src="./assets/images/oracle-logo.png" width="100px" alt=""></li>
+                <li><img src="./assets/images/Google_2015_logo.svg" width="100px" alt=""></li>
+                <li><img src="./assets/images/Mozilla_Foundation_logo_(pre_2017).svg.png" width="100px" alt=""></li>
+                <li><img src="./assets/images/jb_beam.png" width="100px" alt=""></li>
+                <li><img src="./assets/images/Lenovo_logo_2015.svg.png" width="100px" alt=""></li>
+            </ul>
+        </div>
+        <div class="text-center my-2">
+            <h3><i class="fa-solid">&#xf1f9;</i> Copyright
+            <br> Some Rights Reserved
+        </h3>
+        </div>
+    </footer>
```

Comment:
- [ ] Kindly make sure your footer design for both **Home Page** && **About Page** sticks as much as possible to the template design. Your footer is supposed to also include the **logo** so for example once clicked it can immediately lead you to the hero section. Just take look at the **[template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)** and make yours similar to it.

### Template Design
![Screenshot from 2022-11-24 00-18-11](https://user-images.githubusercontent.com/83042742/203655856-b4ae3a38-1f10-487d-aee1-aec832dbdab2.png)

### Your Design
![Screenshot from 2022-11-24 00-02-59](https://user-images.githubusercontent.com/83042742/203653743-787570d9-14bf-4a28-9ef7-ee94cd9299b0.png)